src/libcore/ops.rs

```diff
@@ -470,26 +470,37 @@ rem_impl_float! { f32 f64 }
 ///
 /// # Examples
 ///
-/// A trivial implementation of `Neg`. When `-Foo` happens, it ends up calling
-/// `neg`, and therefore, `main` prints `Negating!`.
+/// An implementation of `Neg` for `Sign`, which allows the use of `-` to
```

Comment:
I would put the comma here, but I also tend to use commas a lot more than most people.